apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -1133,8 +1133,6 @@ const DisplayLocation = ({
     <p className={className}>{locationToDisplay}</p>
   );
 
-Success.isBookingPage = true;
```

Comment:
not needed as it will be handled by `useIsBookingPage()` hook used in `app-providers-app-dir.tsx`